### PR TITLE
Update Czechia holidays: add sk localization

### PR DESCRIPTION
**Pull request by jose1711**
## Proposed change

Add Slovak translation of holidays in Czech Republic.

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [ ] I've added references to all holidays information sources used in this PR
- [ ] The code style looks good: `make pre-commit` command generates no changes
- [ ] All tests pass locally: `make test`, `make tox` (we strongly encourage adding tests to your code)

### CODE REVIEW

File: holidays/locale/sk/LC_MESSAGES/CZ.po [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Czechia holidays sk localization.
+# This file is distributed under the same license as the Python Holidays package.
+# Authors: Jose Riha <jose1711@gmail.com>, (c) 2023.
+msgid ""
+msgstr ""
+"Project-Id-Version: Python Holidays 0.34\n"
```
Comment (arkid15r) [inline holidays/locale/sk/LC_MESSAGES/CZ.po:6]:
```suggestion
"Project-Id-Version: Python Holidays 0.38\n"
```

File: holidays/locale/sk/LC_MESSAGES/CZ.po [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+"POT-Creation-Date: 2023-11-24 00:14+0100\n"
+"PO-Revision-Date: 2023-11-24 00:17+0100\n"
+"Last-Translator: Jose Riha <jose1711@gmail.com>\n"
+"Language-Team: Python Holidays Localization Team\n"
+"Language: sk_SK\n"
```
Comment (arkid15r) [inline holidays/locale/sk/LC_MESSAGES/CZ.po:11]:
```suggestion
"Language: sk\n"
```

**Review by arkid15r — COMMENTED** [review]
LGTM, 
thanks for localizing this!

**Comment by arkid15r** [comment]
> And l10n test needs to be added for new language.

Good point. We normally add tests too.  @jose1711 if you're not familiar with that area you can offload it on me -- I'll submit it in a separate PR

**Comment by jose1711** [comment]
> @jose1711 if you're not familiar with that area you can offload it on me -- I'll submit it in a separate PR

Thank you, I'd appreciate if you take over.

**Comment by arkid15r** [comment]
> Thank you, I'd appreciate if you take over.

You got it!

**Review by arkid15r — APPROVED** [review]

**Pull request merged**